modular/manager/manager.go

```diff
@@ -407,7 +407,7 @@ func (m *ManageModular) FilterGCTask(qTask task.Task) bool {
 }
 
 func (m *ManageModular) FilterUploadingTask(qTask task.Task) bool {
-	return !qTask.Expired() && (qTask.GetRetry() == 0 || qTask.ExceedTimeout())
+	return !qTask.ExceedRetry() && qTask.ExceedTimeout()
```

Comment:
This code logic block picking tasks from Queue. If I use original code, SP works.

Reply:
fixed